### PR TITLE
URDF files for the Chess pieces

### PR DESCRIPTION
**Pull request by Abhay2526**

### CODE REVIEW

**Comment by veds12** [comment]
You exported these files from stl files right?

**Comment by adbidwai** [comment]
Add an image of an example piece in Gazebo simulator too in .png/.jpeg format

File: URDF_Chess_pieces/Black/Bishop_Black/manifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Bishop_Black</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Bishop_Black</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Bishop_Black robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/Black/Bishop_Black/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/Black/Castle_black/manifest.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Castle_black</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Castle_black</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Castle_black robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/Black/Castle_black/manifest.xml]: You can add your email here.

File: URDF_Chess_pieces/Black/Queen_black/manifest.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Queen_black</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Queen_black</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Queen_black robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/Black/Queen_black/manifest.xml]: Same here. Do add your email.

File: URDF_Chess_pieces/Black/Rook_black/manifest.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Rook_black</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Rook_black</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Rook_black robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/Black/Rook_black/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/Black/king_black/manifest.xml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>king_black</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for king_black</p>
+    <p>This package contains configuration data, 3D models and launch files
+for king_black robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/Black/king_black/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/Black/pawn_black/manifest.xml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  <name>pawn_black</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for pawn_black</p>
+    <p>This package contains configuration data, 3D models and launch files
+for pawn_black robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
+  <license>BSD</license>
```
Comment (abhidxt299) [inline URDF_Chess_pieces/Black/pawn_black/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/White/Bishop_white/manifest.xml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Bishop_white</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Bishop_white</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Bishop_white robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/White/Bishop_white/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/White/Castle_white/manifest.xml [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Castle_white</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Castle_white</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Castle_white robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/White/Castle_white/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/White/Queen_white/manifest.xml [hunk 9]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Queen_white</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Queen_white</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Queen_white robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/White/Queen_white/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/White/Rook_white/manifest.xml [hunk 10]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>Rook_white</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for Rook_white</p>
+    <p>This package contains configuration data, 3D models and launch files
+for Rook_white robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/White/Rook_white/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/White/king_white/manifest.xml [hunk 11]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>king_white</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for king_white</p>
+    <p>This package contains configuration data, 3D models and launch files
+for king_white robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/White/king_white/manifest.xml]: You can add your email here

File: URDF_Chess_pieces/White/pawn_white/manifest.xml [hunk 12]
```diff
@@ -0,0 +1,21 @@
+<package format="2">
+  <name>pawn_white</name>
+  <version>1.0.0</version>
+  <description>
+    <p>URDF Description package for pawn_white</p>
+    <p>This package contains configuration data, 3D models and launch files
+for pawn_white robot</p>
+  </description>
+  <author>TODO</author>
+  <maintainer email="TODO@email.com" />
```
Comment (abhidxt299) [inline URDF_Chess_pieces/White/pawn_white/manifest.xml]: You can add your email here

**Review by abhidxt299 — CHANGES_REQUESTED** [review]
The designs are really good.

**Comment by Abhay2526** [comment]
> You exported these files from stl files right?

There was a direct converter to convert the Solidworks file to URDF

**Review by abhidxt299 — APPROVED** [review]
Good job.

**Review by veds12 — APPROVED** [review]
We can merge this PR for now

**Pull request merged**